test/dynamo/test_export.py

```diff
@@ -2009,6 +2009,23 @@ def func(x):
         dynamo_result = exported(inp)
         self.assertTrue(torch._dynamo.utils.same(inp, dynamo_result))
 
+    def test_export_pass_arg_by_name(self):
+        class BasicModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.my_lin = torch.nn.Linear(3, 4, bias=True)
+
+            def forward(self, x):
+                return self.my_lin(x)
+
+        mod, input_tensor = BasicModule(), torch.randn(2, 3)
+        gm, guard = torch._dynamo.export(mod, input_tensor, aten_graph=True)
+        print(gm.graph)
```

Comment:
stray prints

also, this should probably be testing something about the graphs input names?
And can we test the weird corner cases: `*args`, `**kwargs`, etc.